requirements_dev.txt

```diff
@@ -36,20 +36,11 @@ click==8.1.3
     # via
     #   flask
     #   pip-tools
-configargparse==1.5.3
+configargparse==1.5.5
     # via locust
-coreapi==2.3.3
-    # via
-    #   -r ./requirements.txt
-    #   drf-yasg
-coreschema==0.0.4
-    # via
-    #   -r ./requirements.txt
-    #   coreapi
-    #   drf-yasg
```

Comment:
Zijn deze dependencies voor ons obsolete geworden? Met andere woorden, gebruiken we deze niet meer?